modules/core/02-client/types/events.go

```diff
@@ -12,9 +12,11 @@ const (
 	AttributeKeySubjectClientID   = "subject_client_id"
 	AttributeKeyClientType        = "client_type"
 	AttributeKeyConsensusHeight   = "consensus_height"
```

Comment:
should we add a deprecation notice on this attribute key `AttributeKeyConsensusHeight`?